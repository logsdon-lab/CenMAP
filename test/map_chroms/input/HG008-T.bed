version https://git-lfs.github.com/spec/v1
oid sha256:a935a05c1f2ea85c656f4d51023659a673d5856d3d7fbe31f8cb01b6bc5ea451
size 255137
