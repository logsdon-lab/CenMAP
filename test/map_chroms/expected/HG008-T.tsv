version https://git-lfs.github.com/spec/v1
oid sha256:d1061113712ec07530baaf7698d82d06be4b288aae4806d2e4458f027fc9d407
size 1487
